ArtOfIllusion/src/artofillusion/ApplicationPreferences.java

```diff
@@ -26,7 +26,8 @@ Changes copyright (C) 2017 by Maksim Khramov
   private Properties properties;
   private int defaultDisplayMode, undoLevels;
   private double interactiveTol, maxAnimationDuration, animationFrameRate;
-  private boolean keepBackupFiles, useOpenGL, useCompoundMeshTool, reverseZooming, useViewAnimations;
+  private boolean keepBackupFiles, useOpenGL, useCompoundMeshTool, useViewAnimations;
+  private boolean reverseZooming, disableHScroll, reverseHScroll;
   private boolean drawActiveFrustum, drawCameraFrustum, showTravelCuesOnIdle, showTravelCuesScrolling, showTiltDial;
```

Comment:
I feel like we should maybe consider exactly how many customization options we make available. I realize that this is not entirely, or even mostly, due to this PR, but this set of options seems to be getting out of hand, and moving them to their own tab would only be papering over the issue. AOI wants to keep a fairly simple interface. This is not simple.

In particular, I question the `reverseHScroll` option here. I get that there is an ongoing dispute over what "scroll up" and "scroll down" mean, but do you have *any* precedent for people wanting reversed horizontal controls?

Reply:
If there is no separate `reverseHScroll' then the reverse should affect both directions. Basically the wheel with both directions would have the similar logic to a joystick or a 3D-controller. Thumb wheel usage is probably a bit of a rarity and I suppose you'd want both wheels to work the same way anyway. While I was making it adding the check box just seemed the natural thing to do.

Reply:
> moving them to their own tab would only be papering over the issue

The mouse controls are not the reason why I wanted to split the preferences tab. It just adds on the pile. I have been thinking for a long time that it looks confusing as it is. It'd be much clearer to have things that belong together on their own tabs like general/input devices/display.

Reply:
>If there is no separate `reverseHScroll' then the reverse should affect both directions. Basically the wheel with both directions would have the similar logic to a joystick

I've never seen a joystick-reverse-setting swap the H-controls. They're always about the V.

>The mouse controls are not the reason why I wanted to split the preferences tab. It just adds on the pile.

I agree. I'm not even opposed to re-ordering options and adding tabs. I just think that any effort in that direction should include taking a hard look at each of our options, and evaluating whether they are actually adding value to the application.

Reply:
The more I think about it, the harder it is to get my head around *ever* reversing the HScroll Direction. Have you actually seen this done? If not, I think we should leave it out.

If there is actual demand, we can always implement it later.